guides/common/modules/proc_salt-configuring-foreman.adoc

```diff
@@ -0,0 +1,35 @@
+[id="salt_guide_configuring_foreman_{context}"]
+= Configuring {Project}
+
+You need to configure {Project} to use the Salt plugin.
+
+.Procedure
+. Connect to your {Project} using SSH:
++
+[options="nowrap" subs="attributes"]
+----
+# ssh root@{foreman-example-com}
+----
+. Extend the `/etc/sudoers` file to allow the `foreman-proxy` user to run Salt:
++
+[options="nowrap" subs="attributes"]
+----
+Cmd_Alias SALT = /usr/bin/salt, /usr/bin/salt-key
+foreman-proxy ALL = NOPASSWD: SALT
+Defaults:foreman-proxy !requiretty
+----
+. Add a user called `saltuser` to access the Salt API:
++
+[options="nowrap" subs="attributes"]
+----
+# adduser --no-create-home --shell /bin/false --home-dir / saltuser
+# passwd saltuser
+----
++
+Enter the password for the Salt user twice.
++
+[WARNING]
+====
+The command `adduser saltuser -p password` does not work.
+Using it prevents you from importing Salt States.
+====
```

Comment:
Warning: WARNING :-)

Reply:
I've wrapped it in `ifdef`. Is that okay for you?